src/wallet/wallet_rpc_server.cpp

```diff
@@ -3893,7 +3893,26 @@ namespace tools
     crypto::secret_key recovery_val;
     try
     {
-      recovery_val = wal->generate(wallet_file, std::move(rc.second).password(), recovery_key, true, false, false);
+      if (req.enable_multisig_experimental)
+      {
+        // Parse multisig seed into raw multisig data
+        epee::wipeable_string multisig_data;
+        multisig_data.resize(req.seed.size() / 2);
+        if (!epee::from_hex::to_buffer(epee::to_mut_byte_span(multisig_data), req.seed))
+        {
+          er.code = WALLET_RPC_ERROR_CODE_UNKNOWN_ERROR;
+          er.message = "Multisig seed not represented as hexadecimal string";
+          return false;
+        }
+
+        // Generate multisig wallet
+        wal->generate(wallet_file, std::move(rc.second).password(), multisig_data, false);
```

Comment:
I think we also need to set `wal->enable_multisig(true)` as the parameter name suggests, or the wallet's multisig will not be enabled for creating txs and signing.

Reply:
Good point, you could enable it later thru RPC IIRC, but I don't know why you'd want to restore a multisig seed and not enable that flag.